feature/orchestrator/src/main/java/com/simprints/feature/orchestrator/OrchestratorActivity.kt

```diff
@@ -28,13 +37,20 @@ internal class OrchestratorActivity : BaseActivity() {
 
     override fun onStart() {
         super.onStart()
-        val action = intent.action.orEmpty()
-        val extras = intent.extras ?: bundleOf()
-
-        findNavController(R.id.orchestrationHost).setGraph(
-            R.navigation.graph_orchestration,
-            OrchestratorFragmentArgs(action, extras).toBundle()
-        )
+        if(!isGraphInitialized) {
```

Comment:
You could also do this to avoid tracking flags across configuration changes: 
```
with(findNavController(R.id.orchestrationHost)) {
            try {
                graph
            } catch (e: IllegalStateException) {
                val action = intent.action.orEmpty()
                val extras = intent.extras ?: bundleOf()

                setGraph(
                    R.navigation.graph_orchestration,
                    OrchestratorFragmentArgs(action, extras).toBundle()
                )
            }
        }
```

It is probably not better in terms of readability, though :D

Reply:
IMO, a single flag state reads out way better than a try/catch block